src/providers/ldap/sdap_async_groups.c

```diff
@@ -2348,7 +2348,7 @@ static errno_t sdap_nested_group_populate_users(TALLOC_CTX *mem_ctx,
     tmp_ctx = talloc_new(NULL);
     if (!tmp_ctx) return ENOMEM;
 
-    ret = sss_hash_create(tmp_ctx, num_users, &ghosts);
+    ret = sss_hash_create(tmp_ctx, 0, &ghosts);
```

Comment:
It looks like we do know the precise number of entries that will be added to the table. Shouldn't we keep num_users here?

Reply:
Agree.